pkg/rc/fields/fields.go

```diff
@@ -15,6 +15,9 @@ import (
 // string format so that Go will typecheck its uses.
 type ID string
 
+// Strategy is a type alias used for node allocation strategies.
+type Strategy int
```

Comment:
i'd probably rather make this a string so that you can tell what the strategy is by looking at the JSON without needing to correlate it with an enum